pwa/src/templates/documentation/installationDocumentation/InstallationDocumentationTemplate.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from "react";
+import * as styles from "./InstallationDocumentationTemplate.module.css";
+import { Container } from "@conduction/components";
+import { Heading1, Heading2, Heading3, LeadParagraph, Link, Paragraph } from "@gemeente-denhaag/components-react";
+import { ExternalLinkIcon } from "@gemeente-denhaag/icons";
+
+export const InstallationDocumentationTemplate: React.FC = () => {
+  return (
+    <Container layoutClassName={styles.container}>
+      <section className={styles.section}>
+        <Heading1>Componenten Installeren met de Skeleton Applicatie</Heading1>
```

Comment:
Installeren => installeren
Skeleton Applicatie => Skeleton Application óf Skelet Applicatie (I'd probably opt for the English variant).

Reply:
Skeleton Application rewrite goes for all entries within this page. Also, if we're deciding its a "product name" with caps (Skeleton Application instead of skeleton application) we should update this everywhere.